src/Map/Team.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ChessDemo.Map
+{
+    class Team
+    {
+        /// <summary>
+        /// Name of the team
+        /// </summary>
+        public string name;
+
+        /// <summary>
+        /// Team's color.  Currently used for highlighting movement.
+        /// </summary>
+        public Color color;
```

Comment:
For now color is fine, but I'm not sure what it really means in the end. I don't know how to just tint assets easily, at least as well as have them look good